lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+class Menu
+
+attr_reader :menu
+
+def initialize
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for initialize is too high. [18.03/15] (http://c2.com/cgi/wiki?AbcMetric)